devicetypes/smartthings/aeon-led-bulb-6.src/aeon-led-bulb-6.groovy

```
@@ -249,8 +249,8 @@ def setColor(value) {
 		def c = value.hex.findAll(/[0-9a-fA-F]{2}/).collect { Integer.parseInt(it, 16) }
 		result << zwave.switchColorV3.switchColorSet(red: c[0], green: c[1], blue: c[2], warmWhite: 0, coldWhite: 0)
 	} else {
-		def hue = value.hue ?: device.currentValue("hue")
-		def saturation = value.saturation ?: device.currentValue("saturation")
+		def hue = (value.hue == null) ? device.currentValue("hue") : value.hue
```

The Color Control capability is defined as requiring hue and saturation so I don't think we need to guard against null inputs. Also other RGB DTHs like https://github.com/SmartThingsCommunity/SmartThingsPublic/blob/master/devicetypes/smartthings/zigbee-rgbw-bulb.src/zigbee-rgbw-bulb.groovy#L214 don't do these checks. I'd say just remove these null checks and let the errors bubble up like @vseven said.